src/ai/backend/manager/api/exceptions.py

```diff
@@ -414,6 +414,5 @@ class KernelExecutionFailed(BackendAgentError, web.HTTPInternalServerError):
     error_title = 'Executing user code in the kernel has failed.'
 
 
-class UnknownImageReferenceError(BackendError, web.HTTPBadRequest):
-    error_type = 'https://api.backend.ai/probs/unknown-image-reference-error'
-    error_title = 'Unknown Image Reference'
+class UnknownImageReferenceError(ObjectNotFound):
+    object_name = 'Unknown Image Reference'
```

Comment:
The `object_name` field assumes a single noun.
```suggestion
    object_name = 'image reference'
```

Reply:
changed the object_name field according to your opinion.